manifest.yml

```diff
@@ -0,0 +1,35 @@
+name : "AWS_IoT_Device_SDK_for_Embedded_C"       
+version: "202009.00"                              
+description: |-                                  
+  "The AWS IoT Device SDK for Embedded C is a collection of C source 
+  files that can be used in embedded applications to securely connect 
+  to AWS IoT Core and interact with other AWS IoT services on AWS Cloud. \n
+  \n
+  See dependencies for included libraries."         
+dependencies:                                       
+  - name: "coreMQTT"                                
+    version: "1.0.0"                                
+    repository:                                    
+      type: "git"                                  
+      url: "https://github.com/FreeRTOS/coreMQTT"   
+  - name: "coreHTTP"                                
+    version: "1.0.0"                                
+    repository:                                    
+      type: "git"                                  
+      url: "https://github.com/FreeRTOS/coreHTTP"   
+  - name: "coreJSON"                                
+    version: "1.0.0"                               
+    repository:                                    
+      type: "git"                                  
+      url: "https://github.com/FreeRTOS/coreJSON"   
+  - name: "Device-Shadow-for-AWS-IoT-embedded-sdk"  
+    version: "1.0.0"                                
+    repository:                                     
+      type: "git"                                  
+      url: "https://github.com/aws/Device-Shadow-for-AWS-IoT-embedded-sdk"
+  - name: "CMock"                                
+    version: "2.5.2"                             
+    repository:
+      type: "git"                
+      url:  "https://github.com/ThrowTheSwitch/CMock"
+license: "MIT"                                      
```

Comment:
better to add a new line to the end of file.

Reply:
OK. Will do.